includes/class-admin-settings.php

```diff
@@ -671,6 +712,9 @@ public function add_settings_field_callback( $args = [] ) {
 					?>
 				</select>
 				<p>
+					<label for="aspireupdate-settings-field-<?php echo esc_attr( $id ); ?>_other">
+						<?php esc_html_e( 'Enter a custom API host. Ensure that it starts with http:// or https://', 'aspireupdate' ); ?>
```

Comment:
Do you seriously intend to support non-SSL targets? While some users may have a need for that (e.g., local dev without local SSL), I'd think that's a very rare case, and it would be better for most users to simplify the instructions to "starts with https://". That doesn't require changing the actual API; I'd just think that an http endpoint would be the exception case.

Reply:
We do intend to support non-SSL targets, specifically for the exception case you mention. However, I take your point about the description/instructions. There's no need to include http:// there. Removed in [996b347](https://github.com/aspirepress/aspireupdate/pull/384/commits/996b347301985479c06ed1c295d5399f8e3b9462).